docs/user/commandline_tools.rst

```diff
@@ -111,6 +112,20 @@ Get the specified feature type as an encoded feature schema string::
 
     $ geomesa getsft -u username -p password -c test_catalog -f test_feature --format spec
 
+keywords
+~~~~~~~~
+
+Add or remove keywords to a specified schema::
+Repeat the -a or -r flags to add or remove multiple keywords
+The ``-ra``` option removes all keywords
```

Comment:
we removed the short version of the flag - need to update the docs here to --removeAll